app/javascript/lib/scp-api-metrics.js

```diff
@@ -71,12 +74,34 @@ function getFriendlyFilterListByFacet(facets) {
   return filterListByFacet
 }
 
+/** Format parts of query entered by the user to ease analysis in Mixpanel */
+export function formatQueryUserInput(searchedTerms, searchedGenes, searchedFacets) {
```

Comment:
`getLogPropsForSearchParams()` might be a more descriptive name.  At first I thought this was going to generate a url/query string

Reply:
Good point, I've updated this to `getSearchQueryLogProps` in 8d57e81.